src/postprocessing.py

```diff
@@ -37,3 +45,51 @@ def _get_bbox_relative(self, bbox):
         y_max = min(1.0, bbox[3] / w)
         result = [x_min, y_min, x_max, y_max]
         return [str(r) for r in result]
+
+
+class Visualizer(BaseTransformer):
+    def __init__(self, image_size):
+        super().__init__()
+        self.image_size = image_size
+
+    def transform(self, image_ids, results, decoder_dict):
+
+        decoder_dict = decoder_dict
+        image_dir = params.train_imgs_dir
+
+        all_detections = []
+        all_box = []
```

Comment:
@i008 I prefer the `all_detections, all_box(es) = [],[]` and I think we mostly use that around those projects